src/routes/Sidebar.svelte

```diff
@@ -106,7 +106,7 @@ li a {
   <ul slot="navigation" in:fly={{duration: 800, delay: 300, x: -10}}>
     <li><button class:active={currentView === 'current-studies'} on:click={send('current-studies')}>Current Studies</button></li>
     <li>
-      <a class="external" href="/studies">FAQ
+      <a class="external" href="__BASE_SITE__/__FAQ_PATH__">FAQ
```

Comment:
```suggestion
      <a class="external" href="__BASE_SITE__/how-rally-works/faqs/">FAQ
```

nit: this is used exactly once in the code. Is there any advantage in making it a constant vs hardcoding it?